index.html

```diff
@@ -10,7 +10,48 @@
     <link rel="stylesheet" href="styles/index.css" />
 </head>
 <body>
-    <script src="./src/index.js"></script>
+    <header class="header__header">
+        <h1>Weather Report</h1>
+        <span>For the lovely city of 
+            <span id="headerCityName" class="header__city-name"></span></span>
+    </header>
+    <section class="temperature__section">
+        <h2>Temperature</h2>
+        <div class="temperature__content">
+            <div class="temperature__controls">
+                <span id="increaseTempControl">⬆️</span>
+                <span id="tempValue"></span>
+                <span id="decreaseTempControl">⬇️</span>
```

Comment:
In general, use a `button` component for things that are buttons. When you use `span` with a click handler like this, you lose a lot of the accessibility and UI affordance that buttons normally get. Also, because this is a `span`, it participates in horizontal layout and moves around as you click repeatedly. For a vertically-stacked layout like this, next time you might consider putting a column-oriented flexbox on `temperature__controls` and letting the children fill the available space inside this parent `div`.